eternalcore-core/src/main/java/com/eternalcode/core/feature/give/GiveArgument.java

```diff
@@ -15,10 +15,10 @@
 import java.util.List;
 
 @LiteArgument(type = int.class, name = GiveArgument.KEY)
-class GiveArgument extends AbstractViewerArgument<Integer> {
+public class GiveArgument extends AbstractViewerArgument<Integer> {
```

Comment:
If you’re using this argument in two commands, I suggest moving it to the main `argument` package.
Also, I see potential for using this argument in many other places, so I recommend renaming it, for example to `StackAmountArgument` or something similar.